src/connections.ts

```diff
@@ -0,0 +1,106 @@
+import * as VSCode from 'vscode';
+import * as path from 'path';
+
+export function getConnectionStatus(_connectionId: string){
+    const r = Math.random();
+    return r < 0.5 ? 'ok' : 'notok';
+}
```

Comment:
this is a temporary dummy method. It needs to be replaced by actual connection status check